endpoint/endpoint.adef

```diff
@@ -11,13 +11,37 @@ processes:
     }
 }
 
+requires:
+{
+    file:
+    {
+        // needed for curl itself:
+        /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
+        /usr/bin/curl /usr/bin/curl
+
+        // needed for networking:
+        /lib/libnss_compat.so.2 /lib/
+        /lib/libnss_files.so.2 /lib/
+        /lib/libnss_dns.so.2 /lib/
+        /lib/libresolv.so.2 /lib/
+        /etc/nsswitch.conf /etc/
+        /etc/hosts /etc/
+        /etc/resolv.conf /etc/
+
+        /bin/sh /bin/sh
+        /bin/date /bin/date
```

Comment:
Aren't these already existed in `mangOH Red`?

Reply:
The `requires`: section specifies things the component needs from its runtime environment.
We should set the file explicit if we need it.
See https://docs.legato.io/latest/defFilesAdef.html#defFilesAdef_requires can find more information.